src/objects/InstancedMesh.js

```diff
@@ -95,7 +96,7 @@ class InstancedMesh extends Mesh {
 
 		if ( this.instanceColor === null ) {
 
-			this.instanceColor = new BufferAttribute( new Float32Array( this.count * 3 ), 3 );
+			this.instanceColor = new BufferAttribute( new Float32Array( this._maxCount * 3 ), 3 );
```

Comment:
I suggest you do it differently. Instead of introducing `_maxCount`, you can derive the `Float32Array` from the existent `instanceMatrix` buffer attribute.

Reply:
Oh nevermind, i can use ``.count``.